cmd/snap/main.go

```diff
@@ -358,7 +364,7 @@ func run() error {
 				return nil
 			}
 			if e.Type == flags.ErrUnknownCommand {
-				return fmt.Errorf(i18n.G(`unknown command %q, see "snap --help"`), os.Args[1])
+				return fmt.Errorf(i18n.G(`unknown command %q, see "snap help"`), os.Args[1])
```

Comment:
Here is one example of the double quotes with a command inside it. We should pick a pattern and run with it consistently.

Reply:
I found the thing about single vs double quotes.
https://bugs.launchpad.net/snapd/+bug/1659861/comments/1